### ISSUE TITLE
Names of GO terms related to "protection"

### ISSUE THREAD

**Issue by pengyubio** [geneontology/go-ontology#15848]
I see the following GO terms related to "protection".

GO:0090519: anoxia protection
GO:0009650: UV protection
GO:0042262: DNA protection
GO:0048478: replication fork protection

But I think the way that they are written can be ambiguous.

**Comment by pgaudet** [comment]
These should probably be merged into more informative terms? They sound a little like phenotypes.

**Comment by pengyubio** [comment]
What is the distinction between phenotypes and biological processes?

**Comment by pgaudet** [comment]
The biological processes correspond to *normal* cellular/molecular roles of proteins. 
Phenotypes would be what is observed when a gene is mutated, which may or may not directly correspond to a gene's normal function (it may also be a downstream effect). 

Pascale

**Comment by pengyubio** [comment]
If these 4 terms are considered as phenotype, then they should not be included in GO_BP? Or at least more appropriate names should be used to describe the biological processes related to protection?

**Comment by cmungall** [comment]
We should acknowledge that the GO colloquial usage of 'phenotype' is more restricted than the commonly accepted one. All organisms have phenotypes, regardless of what is observed when genes are mutated. See "My mouse has no phenotype":
https://onlinelibrary.wiley.com/doi/epdf/10.1034/j.1601-183X.2002.10201.x

_"A phenotype is a measurable characteristic of the organism one is investigating: every mouse has a phenotype. ‘My mouse has no phenotype’ has become an unfortunate shortcut for saying that a certain mutant has a phenotype that does not deviate from control animals"_

Sorry to be pedantic but I think it's important here for determining what is in scope and what is out of scope.

The definition of GO BP is "a specific objective that the organism is genetically programmed to achieve". (aside: I would still like to tweak this to reflect that it is a process that is executed to achieve some objective that...)

"response to UV"  seems to fit this as it's likely that some organisms have evolved a protective UV response; although the term may cover different mechanisms. Same for "response to oxidative stress" etc

However, the problem is that the response-to-X terms have become a grab bag for representing results of experiments where gene expression is perturbed in response to experimental conditions, or for representing the phenotypic effects of mutation. This is not in the scope of GO. An environmental condition or exposure ontology such as ECTO can be used for the former case and a phenotype or trait ontology for the latter case.

We have over 700 response to terms, some like [response to Aroclor 1254](http://amigo.geneontology.org/amigo/term/GO:1904010) likely don't represent an evolved process. This was for annotating the paper [Gene expression profiles following exposure to a developmental neurotoxicant, Aroclor 1254: pathway analysis for possible mode(s) of action.](http://amigo.geneontology.org/amigo/reference/PMID:18602130).

We have somehow managed to let GO become 4 ontologies: MF, BP, CC and Exposure.

A further problem we have in GO is the duplication in the "cellular response to X" terms. I don't know why we would have both "cellular response to UV" and "response to UV". I believe we should be concerned with the cellular mechanisms only. At the moment it's kind of random which of the two levels are annotated here.

There are a number of things to unpack here and I think this should be a priority for the ontology group. I don't know exactly how to proceed, but maybe something like:

 - Hold off on merges, especially for "proper" BPs like cellular response to oxidative stress
 - Work with the groups who are annotating perturbations in response to exposures to use an exposure ontology, and remove the exposure ontology from GO
 - Implement my strategy for collapsing cellular-X and X
 - Encourage annotation to specific mechanisms; these may be subclasses of a small set of cellular-response-to-X classes, we may want to put do-not-annotate-directly on these

**Comment by ukemi** [comment]
I agree with Chris. We need to think about this carefully. Phenotypes are often used to identify processes that we would want to include in the ontology.

**Comment by pgaudet** [comment]
I agree. My point was that the 'protection' terms were either 'response', or something related to response that should be described more precisely. 

Pascale

**Comment by mah11** [comment]
I don't have grounds for addressing the other terms mentioned in the original summary, but replication fork protection is a specific, active process known by that name in the literature. If the term is renamed, "replication fork protection" should be retained as an exact synonym. (Swapping words around to "protection of replication fork" would make no difference whatsoever to the meaning, and would lose the text match to prevailing usage.)

If it will help to add to the definition, perhaps try something along the lines of:

"A process that preserves the structural integrity of a replication fork that has stalled, preventing fork collapse, degradation of nascent DNA strands, and aberrant fork processing."

cobbled together from PMIDs 26889944, 26403191, 29686033; there's probably scope for further adjustment if you feel motivated

**Comment by pgaudet** [comment]
I think the first 3 terms are relatively straight forward: 
GO:0090519: anoxia protection -> merge into GO:0071454 cellular response to anoxia
GO:0009650: UV protection -> merge into     GO:0034644 cellular response to UV
GO:0042262: DNA protection -> merge into     GO:0006974     cellular response to DNA damage stimulus

(I don't have such a simple solution for fork protection until looking more closely). 

Would these work ? 

Thanks, Pascale

**Comment by tberardini** [comment]
For anoxia protection, the term was created based on this paper:

[Protection of C. elegans from anoxia by HYL-2 ceramide synthase](
https://www.ncbi.nlm.nih.gov/pubmed/19372430)

" Loss of the ceramide synthase gene hyl-2 results in increased sensitivity of C. elegans to anoxia."

@vanaukenk: would you annotate [hyl-2](https://www.wormbase.org/species/c_elegans/gene/WBGene00002044) to 'anoxia protection'?

**Comment by ukemi** [comment]
I also did a little more digging, we might want to have a look at PMID:23435977, PMID:26216848. These two papers describe protection of DNA from damage by a protein that binds to it.

**Comment by ValWood** [comment]
I agree that replication fork protection is a biological process, but I'm all for merging or obsoleting the terms which look like "sensitivities" *.

DNA protection.
Any process in which DNA is protected from damage by, for example, oxidative stress.

This almost certainly is a phenotype. It only has 20 experimental annotations. I recommend obsoleting and rehousing these.

There is a PomBase annotation to SPBC23E6.02 (rrp2), but it is also annotated to repair. 
SGD annotates SRS2 (again repair)
TSA1 peroxiredoxin (upstream detoxification of peroxides)

Do we know of a way to 'protect' DNA against oxidative damage (other than to detoxify free radicals or repair the damage?)

* sensitivities should not even be annotated to "response to x"
because in these cases it really  is only a phenotype (usually inviability), and we know nothing about the actual process which could be  almost any process (transcription, translation, transport, cell cycle, removal of toxicity). In many cases it isn't a "programmed response"  the phenotype is caused by the cell being compromised. Note that nearly all important but non-essential genes have these types of responses to  *many* chemicals. This is why for many geneome over half the annotations are to "response to" terms. My preference would be  to make the "response to..." terms   only avaiable as extensions (occures_during ....in much the same way as we have with phase terms). But this is another issue.

**Comment by ValWood** [comment]
UC protection also only has 31 experimental annotations.

is defined
Any process in which an organism or cell protects itself from ultraviolet radiation (UV), which may also result in resistance to repeated exposure to UV.

So this is about resistance to repeated exposure. I don't know of any yeast process, but maybe 
"melanin production" and such like  in higher euks could be classed as protection? However, this is not how this term has been used. Most of the annotation are to repair enzymes. This would seem to be pathways which repair  UV damage. "UV protection" is not correct here and these could be moved to "response to UV"

"Response to's"  would be better coupled to the *actual* process.
For example:
https://www.uniprot.org/uniprot/A3KMN2 (ERCC6)
BP "pyrimidine dimer repair"  occurs_during "response to UV"

**Comment by ValWood** [comment]
For the UC protection annotations 

GO:0070914   UV-damage excision repair

might be the correct term. It's a bit of a problem in this branch because sometimes pathways are described by mechanism, and sometimes by damage causing agent.


"base/nuceotide exision repair" occurs_during "response to UV (damage)"

would improve annotation specificity and consistency

**Comment by ValWood** [comment]
PomBAse annotation to "DNA protection" corrected to "DNA repair"
